helix-view/src/document.rs

```diff
@@ -88,7 +88,7 @@ pub struct Document {
 
     syntax: Option<Syntax>,
     // /// Corresponding language scope name. Usually `source.<lang>`.
-    pub(crate) language: Option<Arc<LanguageConfiguration>>,
+    pub language: Option<Arc<LanguageConfiguration>>,
```

Comment:
Not sure if this is a good idea?

Reply:
Use `doc.language()`

Reply:
Thanks, totally missed that there is `language_config()`. Done.